Sources/Build/BuildPlan.swift

```diff
@@ -1313,3 +1321,10 @@ struct SwiftBackDeployLibrariesNote: DiagnosticData {
             $0 <<< "Swift compiler no longer supports statically linking the Swift libraries. They're included in the OS by default starting with macOS Mojave 10.14.4 beta 3. For macOS Mojave 10.14.3 and earlier, there's an optional Swift library package that can be downloaded from \"More Downloads\" for Apple Developers at https://developer.apple.com/download/more/"
     })
 }
+
+private extension ResolvedTarget {
+    var dependsOnAnySystemModules: Bool {
+        let dependsOnAnySystemModules = !dependencies.filter { $0.target?.dependsOnAnySystemModules == true }.isEmpty
+        return dependsOnAnySystemModules || type == .systemModule
```

Comment:
`type == .systemModule` is unnecessary since they don't have any code.

Reply:
wait, that's the whole point of the method, no? Without that it'll always return false